CONTRIBUTING.tr.md

```diff
@@ -74,12 +74,42 @@ Github, AGC assembly dili için sentaks desteği sağlıyor ama sizin kodu düze
 
 ## Neyi kontrol edeyim?
 
-Taratılmış çıktılar ve bu repodaki kaynak kodu arasındaki aşağıdakileri kapsayan herhangi bir tutarsızlığı:
+Taratılmış çıktılar ve bu repodaki kaynak kodu arasındaki herhangi bir tutarsızlığı.
 
 ### Yorumlar
 
-- Uyarlanan koddaki yorum satırları, çıktılar ile tamamen eşleşmelidir
-  - Bu bilerek yazım hatası koymayı veya bir yorumun tamamını kaldırıp/eklemeyi içerebilir.
+Uyarlanan koddaki yorum satırları, çıktılar ile **tamamen** **eşleşmelidir**.
+
+Düzeltmek için bakmanız gereken bazı temel sorunlar şunlardır:
+
+#### Yazım Hataları
+
+Bazı yerlerde, orijinal geliştiriciler yorum yazarken yazım hataları yapmış. Bunların bazıları dijitalleştirme sürecinin başlarında yanlışlıkla düzeltildi fakat dijitalleştirme süreci taratılmış çıktılarda olmayan yeni yazım hatalarına da sebep oldu.
+
+Örneğin, dijitalleştirilmiş yorumlar `SPACECRAFT` kelimesini içeriyorsa ama taratılmış çıktılarda `SPAECRAFT` yazıyorsa, dijitalleştirilmiş yorum `SPAECRAFT` (`C` eksik) olarak **düzeltilmelidir**.
+
+Aynı şekilde, eğer dijitalleştirmiş yorumlarda bir yazım hatası varsa ama taratılmış çıktılarda doğru yazılmışsa, yazım hatası **düzeltilmelidir**.
+
+### Boşluklar
+
+- Bir string'deki iki karakterin arasındaki boşluklar şu kurallara uymalıdır ([#316][10]'daki tartışmayı inceleyin):
+  - Yeni kelimeler arası tek boşluk.
+  - Yeni cümleler arası iki boşluk.
+  - Yeni girintiler için üç boşluk.
+
+Örneğin bu:
+
+```plain
+	1)  FOO BAR BAZ QUX QUUX QUUZ. CORGE, GRAULT,
+	GARPLY, WALDO.
+```
+
+Buna dönüşmeli:
+
+```plain
+	1) FOO BAR BAZ QUX QUUX QUUZ.  CORGE, GRAULT,
+	   GARPLY, WALDO.
+```
```

Comment:
This example was replaced by:

> Not all pages in the scans follow this generalisation, if the scans only have a single space instead of a double space, use a single space.

Reply:
Fixed it in #89dac5fc497f4debe6517b54a582c20e7b8e1114